yarn-project/boxes/private-token/src/tests/sandbox.test.ts

```diff
@@ -0,0 +1,80 @@
+import { AccountWallet, AztecAddress, CompleteAddress, Contract, Fr, Wallet } from '@aztec/aztec.js';
+import { createDebugLogger } from '@aztec/foundation/log';
+import { PrivateTokenContract } from '../artifacts/PrivateToken.js';
+import { rpcClient } from '../config.js';
+import { deployContract } from '../scripts/deploy_contract.js';
+import { getWallet } from '../scripts/util.js';
+
+const logger = createDebugLogger('aztec:http-rpc-client');
+
+function getRandomInt(min: number, max: number): bigint {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return BigInt(Math.floor(Math.random() * (max - min)) + min);
+}
+
+const MIN_INITIAL_BALANCE = 300;
+const MAX_INITIAL_BALANCE = 600;
+const INITIAL_BALANCE = getRandomInt(MIN_INITIAL_BALANCE, MAX_INITIAL_BALANCE);
```

Comment:
randomizing here to avoid duplicate tranactions, in the case we want to test repeatedly against the same devnet without tearing down the sandbox each time

Reply:
There shouldn't be duplicate transactions even when this value is fixed. Instead, make the salt a random number.